src/helpers/storage.js

```diff
@@ -22,12 +22,11 @@
  * @returns {Array}
  */
 export const getDeviceAncestors = (deviceData, device) => {
-    // device ancestors including the device itself
     const ancestors = [];
     const deviceParents = deviceData[device]?.parents?.v || [];
 
-    ancestors.push(device);
     deviceParents.forEach(parent => {
+        ancestors.push(parent);
```

Comment:
The impact of this change isn't obvious from just the patch, it depends on where else `deviceParents` is used. If you checked this, ok.

Reply:
Yeap I checked it.